src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -47,6 +47,52 @@ <h4 *ngIf="course.id" id="jhi-course-heading-edit" jhiTranslate="artemisApp.cour
                         </div>
                     </ng-container>
                 </div>
+                <div class="form-check">
+                    <input type="checkbox" id="field_maxComplaintSettingEnabled" (change)="changeEvent($event)" />
```

Comment:
this check should actually enable / disable both textfields for max number of complaints and the max duration in days. In the UI it looks like it only covers the first value